lib/streamlit/config.py

```diff
@@ -448,6 +448,19 @@ def _logger_message_format() -> str:
     type_=bool,
 )
 
+_create_option(
+    "runner.fastReruns",
+    description="""
+        Handle script rerun requests immediately, rather than waiting for
+        script execution to reach a yield point. Enabling this will
+        make Streamlit much more responsive to user interaction, but it can
+        lead to race conditions in apps that mutate session_state data outside
+        of explicit session_state assignment statements.
+    """,
+    default_val=True,
```

Comment:
I agree that it should default to False until we're confident everything works correctly and it is safe to have on by default